backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -156,6 +160,7 @@ type Handler struct {
 	waitForStartChan (chan WaitForStartReceive)
 	selectTopicChan  (chan WordSelectReceive)
 	playTimeChan     (chan PlayTimeChanReceive)
+	EndChan          (chan bool)
```

Comment:
Lol these sound like japanese girls, `end-chan`. Any particular reason this is public but the rest are private?

Reply:
Good catch. This should be private